deploy/terraform/bootstrap/sap_library/output.tf

```
@@ -26,11 +26,19 @@ output "fileshare_sapbits_name" {
   value     = module.sap_library.fileshare_sapbits_name
 }
 
-output "user_vault_name" {
+output "library_user_key_vault_name" {
   sensitive = true
   value     = module.sap_library.user_vault_name
 }
```

The value needs to be changed accordingly :)
```suggestion
output "library_user_key_vault_name" {
  sensitive = true
  value     = module.sap_library. library_user_key_vault_name
}
```

---

Good catch! Thanks! Updated.